### DISCUSSION TITLE
Expose a button service call for remote start to Siri/Homekit

### DISCUSSION THREAD

**Discussion by mbellis18** [Hyundai-Kia-Connect/kia_uvo#573] [Q&A]
I created a button for a call service to remote start my telluride on the dashboard.  Is it possible. to expose this to the homekit integration so that I can use siri to remote start the car?  Thanks in advance!

**Comment by cdnninja** [comment]
This is more of a homekit / HA question.  The way I did it with google home was a switch with my specific service call.  After the service call is sent my automation switches it back off.   Quick google shows homekit is probably the same.

**Comment by Jyates18** [comment]
I only have a lock for my doors on telluride, and a bunch of sensors and binary sensors. no switch for remote start, how'd you set this up?

**Reply by jkuffner** [reply]
Did you get this working? I just installed the integration that all I seem to have is the ability to lock and unlock the doors on my Sorrento.

**Reply by cdnninja** [reply]
> Did you get this working? I just installed the integration that all I seem to have is the ability to lock and unlock the doors on my Sorrento.

It isn't on the entity but rather a service call.  Go to developer tools on the left.  Then actions up top.   Then select the call you want as per the readme.  

Can tie those to automations as well.

**Comment by mgguinne** [comment]
I just created a script in HA and used the ios HA shortcut action to call the script. It works well from Siri to turn on my air conditioning before I am ready to leave home.
@mbellis18
